packages/config/src/log/cleanup.js

```diff
@@ -93,6 +93,17 @@ const simplifyConfig = function ({
   const buildA = {
     ...build,
     ...removeEmptyArray(environment, 'environment'),
+    ...removeEmptyObject(
+      {
+        ...processing,
+        ...removeEmptyObject(css, 'css'),
+        ...removeEmptyObject(html, 'html'),
+        ...removeEmptyObject(images, 'images'),
+        ...removeEmptyObject(js, 'js'),
+      },
+      'processing',
+    ),
+    ...removeEmptyObject(services, 'services'),
```

Comment:
Those lines ensure that empty objects used as default values are not logged in verbose mode.